keep-ui/app/alerts/alert-history-charts.tsx

```diff
@@ -36,9 +36,7 @@ export default function AlertHistoryCharts({
     timeUnit = "Hours";
   }
 
-  const alertsReversed = [...alerts].reverse();
-
-  const rawChartData = alertsReversed.reduce((prev, curr) => {
+  const rawChartData = [...alerts].reverse().reduce((prev, curr) => {
```

Comment:
If you use [`toReversed()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toReversed), you don't need to copy arrays.